### ISSUE TITLE
CommunicationError : Error while copying content to a stream. / Cannot open page 130455

### ISSUE THREAD

**Issue by Clusterduck1** [microsoft/navcontainerhelper#858]
I am not able to run tests in my BC container. Neither ```Run-TestsInBCContainer``` nor ```Get-TestsFromBCContainer``` do work.

My script for creating a new BC Contianer (test toolkit included):
```
function CreateNewBCContainer() {
    [CmdletBinding()]
    param (
        [Parameter(Mandatory = $true)]
        [string] $LicenseFile,
        [Parameter(Mandatory = $false)]
        [string] $ImageName,
        [Parameter(Mandatory = $false)]
        [string] $ContainerName
    )

    # install Cronus (DE) if not specified
    if ([string]::IsNullOrEmpty($imageName)) {
        $ImageName = 'mcr.microsoft.com/businesscentral/onprem:de-ltsc2019'
    }

    # set default container name
    if ([string]::IsNullOrEmpty($containerName)) {
        $ContainerName = 'navserver'
    }
    
    [pscredential] $credential = New-Object -TypeName pscredential -ArgumentList 'admin', (ConvertTo-SecureString 'Password1' -AsPlainText -Force)

    # install navcontainerhelper
    if (-not (Get-Module -ListAvailable -Name navcontainerhelper)) {
        Install-Module navcontainerhelper -Force
    } else {
        Update-Module navcontainerhelper -Force
    }

    New-BCContainer -licenseFile $LicenseFile `
        -imageName $ImageName `
        -containerName $ContainerName `
        -auth NavUserPassword `
        -Credential $credential `
        -accept_eula `
        -assignPremiumPlan `
        -accept_outdated `
        -doNotCheckHealth `
        -shortcuts Desktop `
        -updateHosts `
        -includeTestToolkit
}
```
While executing
```
[pscredential] $credential = New-Object -TypeName pscredential -ArgumentList 'admin', (ConvertTo-SecureString 'Password1' -AsPlainText -Force)

Get-TestsFromBCContainer -containerName Cronus -credential $credential -testSuite 'DEFAULT' -companyName ((Get-CompanyInBCContainer -containerName Cronus).CompanyName)
```
I do get this error message:
```
CommunicationError : Error while copying content to a stream.
at <ScriptBlock>, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\ClientContext.ps1: line 77
at InvokeInteractionAndCatchForm, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\ClientContext.ps1: line 218
at OpenForm, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\ClientContext.ps1: line 229
at Get-Tests, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\PsTestFunctions.ps1: line 152
at <ScriptBlock>, <No file>: line 51
Current Interaction: Microsoft.Dynamics.Framework.UI.Client.Interactions.OpenFormInteraction
Time spend: 0 seconds
Cannot open page 130455. You might need to import the test toolkit to the container and/or remove the folder C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4 and retry. You might also have URL or Company name wrong.
at Get-Tests, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\PsTestFunctions.ps1: line 154
at <ScriptBlock>, <No file>: line 51
Cannot open page 130455. You might need to import the test toolkit to the container and/or remove the folder C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4 and retry. You might also have URL or Company name wrong.
In C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.6.4.26\ContainerHandling\Invoke-ScriptInNavContainer.ps1:37 Zeichen:13
+             Invoke-Command -Session $session -ScriptBlock $scriptbloc ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Cannot open pag...any name wrong.:String) [], RuntimeException
    + FullyQualifiedErrorId : Cannot open page 130455. You might need to import the test toolkit to the container and/or remove the folder C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4 and retry. You might also have URL or Company name wrong.
```

**Comment by freddydk** [comment]
If you could share the output of your container creation - that contains a lot of debug info, which is very useful.

**Comment by Clusterduck1** [comment]
Of course. I just obfuscated two paths.
```
C:\******\NewBCContainer.ps1
NavContainerHelper is version 0.6.4.26
NavContainerHelper is running as administrator
Host is Microsoft Windows 10 Pro - 1909
Docker Client Version is 19.03.5
Docker Server Version is 19.03.5
Using image mcr.microsoft.com/businesscentral/onprem:de-ltsc2019
Disabling Health Check (always report healthy)
Removing container Cronus
Removing Cronus from host hosts file
Removing C:\ProgramData\NavContainerHelper\Extensions\Cronus
Creating Container Cronus
Version: 15.0.36560.0-de
Style: onprem
Platform: 15.0.36510.0
Generic Tag: 0.0.9.95
Container OS Version: 10.0.17763.737 (ltsc2019)
Host OS Version: 10.0.18363.592 (1909)
The container operating system does not match the host operating system, forcing hyperv isolation.
Using locale de-DE
Using hyperv isolation
Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
Using license file C:\******.flf
Files in C:\ProgramData\NavContainerHelper\Extensions\Cronus\my:
- AdditionalOutput.ps1
- license.flf
- MainLoop.ps1
- SetupNavUsers.ps1
- SetupVariables.ps1
- SetupWebClient.ps1
- updatehosts.ps1
Creating container Cronus from image mcr.microsoft.com/businesscentral/onprem:de-ltsc2019
1e8d2f7d0757c804171bb35e5b0e8f3a893d499c6ed2830f092da9fcbbbce8ce
Waiting for container Cronus to be ready
Initializing...
Setting host.docker.internal to  in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to  in container hosts file (copy from host hosts file)
Setting host.docker.internal to 192.168.178.30 in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to 192.168.178.30 in container hosts file (copy from host hosts file)
Setting kubernetes.docker.internal to 127.0.0.1 in container hosts file (copy from host hosts file)
Setting host.containerhelper.internal to 172.30.208.1 in container hosts file
Starting Container
Hostname is Cronus
PublicDnsName is Cronus
WARNING: Container starts with TimeZone = MitteleuropÃ¤ische Zeit, which is not recognized in the list of TimeZones.
Using NavUserPassword Authentication
Starting Local SQL Server
Starting Internet Information Server
Creating Self Signed Certificate
Self Signed Certificate Thumbprint 57E727C63AF53782C2C53281A5FE8EF34C2F892F
Modifying Service Tier Config File with Instance Specific Settings
Starting Service Tier
Registering event sources
Creating DotNetCore Web Server Instance
Using license file 'c:\run\my\license.flf'
Import License
Creating http download site
Setting SA Password and enabling SA
Creating admin as SQL User and add to sysadmin
Creating SUPER user
Assign Premium plan for ADMIN
Container IP Address: 172.30.217.150
Container Hostname  : Cronus
Container Dns Name  : Cronus
Web Client          : http://Cronus/BC/
Dev. Server         : http://Cronus
Dev. ServerInstance : BC
Setting Cronus to 172.30.217.150 in host hosts file

Files:
http://Cronus:8080/al-4.0.176004.vsix

WARNING: You are running a container which is 159 days old.
Microsoft recommends that you always run the latest version of our containers.

Initialization took 99 seconds
Ready for connections!
Reading CustomSettings.config from Cronus
Creating Desktop Shortcuts for Cronus
Copy from container Cronus (C:\Applications\TestFramework\TestLibraries\Any\Microsoft_Any.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Any.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Any.app
Synchronizing Any on tenant default
Installing Any on tenant default
App successfully published
Copy from container Cronus (C:\Applications\TestFramework\TestLibraries\Assert\Microsoft_Library Assert.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Library Assert.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Library Assert.app
Synchronizing Library Assert on tenant default
Installing Library Assert on tenant default
App successfully published
Copy from container Cronus (C:\Applications\TestFramework\TestRunner\Microsoft_Test Runner.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Test Runner.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Test Runner.app
Synchronizing Test Runner on tenant default
Installing Test Runner on tenant default
App successfully published
Copy from container Cronus (C:\Applications\System Application\Test\Microsoft_System Application Test Library.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_System Application Test Library.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_System Application Test Library.app
Synchronizing System Application Test Library on tenant default
Installing System Application Test Library on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-TestLibraries.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-TestLibraries.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-TestLibraries.app
Synchronizing Tests-TestLibraries on tenant default
Installing Tests-TestLibraries on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Bank.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Bank.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Bank.app
Synchronizing Tests-Bank on tenant default
Installing Tests-Bank on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Cash Flow.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Cash Flow.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Cash Flow.app
Synchronizing Tests-Cash Flow on tenant default
Installing Tests-Cash Flow on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Cost Accounting.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Cost Accounting.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Cost Accounting.app
Synchronizing Tests-Cost Accounting on tenant default
Installing Tests-Cost Accounting on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-CRM integration.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-CRM integration.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-CRM integration.app
Synchronizing Tests-CRM integration on tenant default
Installing Tests-CRM integration on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Data Exchange.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Data Exchange.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Data Exchange.app
Synchronizing Tests-Data Exchange on tenant default
Installing Tests-Data Exchange on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Dimension.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Dimension.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Dimension.app
Synchronizing Tests-Dimension on tenant default
Installing Tests-Dimension on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-ERM.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-ERM.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-ERM.app
Synchronizing Tests-ERM on tenant default
Installing Tests-ERM on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Fixed Asset.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Fixed Asset.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Fixed Asset.app
Synchronizing Tests-Fixed Asset on tenant default
Installing Tests-Fixed Asset on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-General Journal.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-General Journal.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-General Journal.app
Synchronizing Tests-General Journal on tenant default
Installing Tests-General Journal on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Graph.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Graph.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Graph.app
Synchronizing Tests-Graph on tenant default
Installing Tests-Graph on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Integration.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Integration.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Integration.app
Synchronizing Tests-Integration on tenant default
Installing Tests-Integration on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Invoicing.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Invoicing.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Invoicing.app
Synchronizing Tests-Invoicing on tenant default
Installing Tests-Invoicing on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Job.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Job.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Job.app
Synchronizing Tests-Job on tenant default
Installing Tests-Job on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Local.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Local.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Local.app
Synchronizing Tests-Local on tenant default
Installing Tests-Local on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Misc.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Misc.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Misc.app
Synchronizing Tests-Misc on tenant default
Installing Tests-Misc on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Permissions.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Permissions.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Permissions.app
Synchronizing Tests-Permissions on tenant default
Installing Tests-Permissions on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Physical Inventory.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Physical Inventory.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Physical Inventory.app
Synchronizing Tests-Physical Inventory on tenant default
Installing Tests-Physical Inventory on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Prepayment.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Prepayment.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Prepayment.app
Synchronizing Tests-Prepayment on tenant default
Installing Tests-Prepayment on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Rapid Start.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Rapid Start.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Rapid Start.app
Synchronizing Tests-Rapid Start on tenant default
Installing Tests-Rapid Start on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Report.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Report.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Report.app
Synchronizing Tests-Report on tenant default
Installing Tests-Report on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Resource.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Resource.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Resource.app
Synchronizing Tests-Resource on tenant default
Installing Tests-Resource on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Reverse.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Reverse.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Reverse.app
Synchronizing Tests-Reverse on tenant default
Installing Tests-Reverse on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-SCM.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-SCM.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-SCM.app
Synchronizing Tests-SCM on tenant default
Installing Tests-SCM on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-SMB.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-SMB.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-SMB.app
Synchronizing Tests-SMB on tenant default
Installing Tests-SMB on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-SMTP.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-SMTP.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-SMTP.app
Synchronizing Tests-SMTP on tenant default
Installing Tests-SMTP on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Upgrade.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Upgrade.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Upgrade.app
Synchronizing Tests-Upgrade on tenant default
Installing Tests-Upgrade on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-User.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-User.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-User.app
Synchronizing Tests-User on tenant default
Installing Tests-User on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-VAT.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-VAT.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-VAT.app
Synchronizing Tests-VAT on tenant default
Installing Tests-VAT on tenant default
App successfully published
Copy from container Cronus (C:\Applications\BaseApp\Test\Microsoft_Tests-Workflow.app) to C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Workflow.app
Publishing C:\ProgramData\NavContainerHelper\Extensions\Cronus\_Microsoft_Tests-Workflow.app
Synchronizing Tests-Workflow on tenant default
Installing Tests-Workflow on tenant default
App successfully published
TestToolkit successfully imported
Container Cronus successfully created
```

**Comment by freddydk** [comment]
I have run the exact script on my machine - it works fine.
Do you have antivirus software installed that could cause this maybe?

**Comment by Clusterduck1** [comment]
Thank you for helping me. Yes we are running TrendMicro OfficeScan here. Do you think that could cause the trouble? What about the log warnings - like  "WARNING: Container starts with TimeZone = MitteleuropÃ¤ische Zeit, which is not recognized in the list of TimeZones." do you think they could be the cause?

**Comment by freddydk** [comment]
I don't think it is the timezone issue.
If you temporarily could try to disable the virus scanner?

**Comment by freddydk** [comment]
Actually - I think you are right - looking at other people with problems with test running also has a strange timezone. Will try to create an Azure VM which is not English and test.

**Comment by Clusterduck1** [comment]
I had to consult my colleague from the IT department for this but unfortunately that didnt help.
This is the output I get.

```
CommunicationError : Error while copying content to a stream.
at <ScriptBlock>, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\ClientContext.ps1: line 77
at InvokeInteractionAndCatchForm, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\ClientContext.ps1: line 218
at OpenForm, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\ClientContext.ps1: line 229
at Get-Tests, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\PsTestFunctions.ps1: line 152
at <ScriptBlock>, <No file>: line 51
Current Interaction: Microsoft.Dynamics.Framework.UI.Client.Interactions.OpenFormInteraction
Time spend: 1 seconds
Cannot open page 130455. You might need to import the test toolkit to the container and/or remove the folder C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4 and retry. You might also have URL or Company name wrong.
at Get-Tests, C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4\PsTestFunctions.ps1: line 154
at <ScriptBlock>, <No file>: line 51
Cannot open page 130455. You might need to import the test toolkit to the container and/or remove the folder C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4 and retry. You might also have URL or Company name wrong.
In C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.6.4.26\ContainerHandling\Invoke-ScriptInNavContainer.ps1:37 Zeichen:13
+             Invoke-Command -Session $session -ScriptBlock $scriptbloc ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Cannot open pag...any name wrong.:String) [], RuntimeException
    + FullyQualifiedErrorId : Cannot open page 130455. You might need to import the test toolkit to the container and/or remove the folder C:\ProgramData\NavContainerHelper\Extensions\Cronus\PsTestTool-4 and retry. You might also have URL or Company name wrong.
```

**Comment by freddydk** [comment]
The fix for this is https://github.com/microsoft/navcontainerhelper/commit/c49d0b669825f126c0e3816976cbabc17fc931ed

**Comment by freddydk** [comment]
Will ship in the next version of ContainerHelper (later today)

**Comment by freddydk** [comment]
Fixed in 0.6.4.27
